crew

```diff
@@ -739,27 +740,20 @@ def resolve_dependencies
 
   return if @dependencies.empty?
 
+  puts "\nThe following packages also need to be installed:\n\n"
 
-  puts 'The following packages also need to be installed: '
-
-  i = 0
-  last_deps = []
-  last_packages = ["curl", "ghc", "mandb", "gtk3", "sommelier"]
+  last_packages = ['curl', 'ghc', 'mandb', 'gtk3', 'sommelier']
```

Comment:
We can remove `curl` and `mandb` since they are already core packages.

Reply:
Done